### PR TITLE
GH53296 OKD Updating clusters section needs to be updated

### PR DESCRIPTION
**Pull request by mburke5678**
Editing updating docs for OKD.

Mostly structural changes, ifdef, and a few changes to the text.

**Hidden assemblies** 
[Understanding OpenShift updates](https://docs.openshift.com/container-platform/4.12/updating/understanding-openshift-updates.html) (Mostly about OSUS, not in [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/index.html))

[Updating a cluster that includes RHEL compute machines](https://docs.openshift.com/container-platform/4.12/updating/updating-cluster-rhel-compute.html). N/A in [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-restricted-network-cluster/index.html)

[Updating a cluster in a disconnected environment using the OpenShift Update Service](https://docs.openshift.com/container-platform/4.12/updating/updating-restricted-network-cluster/restricted-network-update-osus.html) 

[Uninstalling the OpenShift Update Service from a cluster](https://docs.openshift.com/container-platform/4.12/updating/updating-restricted-network-cluster/uninstalling-osus.html) N/A in [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-restricted-network-cluster/index.html)

[Preparing to perform an EUS-to-EUS update](https://docs.openshift.com/container-platform/4.12/updating/preparing-eus-eus-upgrade.html) N/A in [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/index.html)

**Modified:**
[index/Understanding upgrade channels and releases](https://docs.openshift.com/container-platform/4.12/updating/index.html#updating-clusters-overview-upgrade-channels-and-releases). Removed link and references in [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates//updating/index.html#updating-clusters-overview-upgrade-channels-and-releases).

[Understanding upgrade channel](https://docs.openshift.com/container-platform/4.12/updating/understanding-upgrade-channels-release.html). Only one channel for [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/index.html).

Updating a cluster by using the web console. Added ifdef to switch between `stable-Branch Build` ([OCD](https://58413--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html#update-upgrading-web_updating-cluster-within-minor)) and `stable-4` ([OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-cluster-within-minor.html#update-upgrading-web_updating-cluster-within-minor)) 

Changing the update server by using the web console, Added ifdefs to remove [the second sentence on OSUS](https://58413--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html#update-changing-update-server-web_updating-cluster-within-minor) from [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-cluster-within-minor.html#update-changing-update-server-web_updating-cluster-within-minor).

Understanding OpenShift Container Platform update duration. Added ifdefs to remove [Red Hat Enterprise Linux (RHEL) compute nodes module](https://58413--docspreview.netlify.app/openshift-enterprise/latest/updating/understanding-openshift-update-duration.html#redhat-enterprise-linux-nodes_openshift-update-duration) from [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates//updating/understanding-openshift-update-duration.html#redhat-enterprise-linux-nodes_openshift-update-duration).

[Updating a cluster in a disconnected environment without the OpenShift Update Service](https://58413--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html). Used ifdefs to remove _without the OpenShift Update Service_ from [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-restricted-network-cluster/restricted-network-update.html)  

[Updating a cluster in a disconnected environment without the OpenShift Update Service](https://docs.openshift.com/container-platform/4.12/updating/updating-restricted-network-cluster/restricted-network-update.html) Changed [name of assembly in OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-restricted-network-cluster/restricted-network-update.html).

**Added:** 
[Updating a cluster in a disconnected environment](https://58413--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/index.html). Was not included in [OKD](http://file.rdu.redhat.com/mburke/GH53296-OKD-updating-section-updates/updating/updating-restricted-network-cluster/index.html) upon initial publication.

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@binnes PTAL

**Review by vrutkovs — APPROVED** [review]

**Comment by binnes** [comment]
Is there a rendered version I can see?  The file change log looks OK, but difficult to verify without seeing a rendered version of the updated content.

File: updating/index.adoc [hunk 1]
```diff
@@ -8,29 +8,43 @@ toc::[]
...
 * xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding fast and stable channel use and strategies]
 * xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
 * xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
 * xref:../updating/understanding-upgrade-channels-release.adoc#conditional-updates-overview_understanding-upgrade-channels-releases[Understanding conditional updates]
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+With upgrade channels, you can choose an upgrade strategy. Upgrade channels are specific to a minor version of {product-title}. Upgrade channels only control release selection and do not impact the version of the cluster that you install. The `openshift-install` binary file for a specific version of the {product-title} always installs that minor version.
+
```
Comment (mburke5678) [inline updating/index.adoc:30]: Peer review: I did not change this paragraph from [Line 20](https://github.com/openshift/openshift-docs/pull/58413/files#diff-24163360618968f63d32275a3f76bdddaf7f4d60ac9ef56a8a51433ea9b8cd87R20), except to remove the link.

**Review by mjpytlak — COMMENTED** [review]
@mburke5678 Your changes LGTM. And thank you for the detailed notes and links on the previews between OCP and OKD. Made it easy to verify your changes.

**Comment by mburke5678** [comment]
@binnes I have merged these changes. Please feel free to suggest any other changes I might have missed.

**Pull request merged**